### PR TITLE
Update Hardware.rst

### PR DESCRIPTION
**Pull request by gyto6**
Repair website display and adding comment

### CODE REVIEW

File: docs/Performance and Tuning/Hardware.rst
Comment (gmelikov) [inline docs/Performance and Tuning/Hardware.rst:716]: What do you think about removing all additions per model, and this paragraph too (it became stale after your PR). So, looks like all these models may be great and people should choose based on their budget.

Comment (gyto6) [reply docs/Performance and Tuning/Hardware.rst:716]: Indeed, that's why'd prefer, but I tryed to stay humble in my contribution. 

I'll remove the comments and try to update the following paragraph to share your words.

Comment (gmelikov) [reply docs/Performance and Tuning/Hardware.rst:716]: I'm ok to just remove this paragraph, jfyi

**Comment by gyto6** [comment]
I removed all, and checked the preview, everything is fine this time concerning the shape.

**Comment by gyto6** [comment]
Ready to merge. Waiting for approval or remarks.

File: docs/Performance and Tuning/Hardware.rst
Comment (gmelikov) [inline docs/Performance and Tuning/Hardware.rst:712]: This paragraph is still here, my thought was to drop it (we have more than one model now)

Comment (gyto6) [reply docs/Performance and Tuning/Hardware.rst]: Wasn't sure, I deleted the whole paragraph this time

**Review by gmelikov — APPROVED** [review]

**Pull request merged**
